tests/broccoli/glimmer-app-test.ts

```diff
@@ -284,19 +286,26 @@ describe('glimmer-app', function() {
         'config': {},
       });
 
-      let app = createApp() as any;
-      let output = await buildOutput(app.cssTree());
+      let app = createApp({
+        trees: {
+          nodeModules: path.join(__dirname, '..', '..', '..', 'node_modules')
+        }
+      }) as any;
+      let output = await buildOutput(app.toTree());
+      let actual = output.read();
 
-      expect(output.read()).to.deep.equal({
-        'app.css': `body {\n  font: 100% Helvetica, sans-serif;\n  color: #333; }\n`,
-      });
+      expect(actual['app.css']).to.equal(
```

Comment:
I had to switch to `app.toTree()` instead of `app.cssTree()` I don't really remember why I had to do this.
I'd guess because we have to run more code now to compile the scss (with ember-cli-sass).
This means we need an `index.ts` and `index.html` file to build..

I'm not sure how else I could test this easily